docs/source/quickstart.rst

```diff
@@ -10,11 +10,78 @@ Setup
 
 For a more detailed guide on installation options please refer to the :doc:`install` page.
 
+Send/Recv NumPy Arrays
+---------------------
+
+Process 1 - Server
+~~~~~~~~~~~~~~~~~~
+
+.. code-block:: python
+
+    import asyncio
+    import ucp
+    import numpy as np
+
+    n_bytes = 2**30
+    host = ucp.get_address(ifname='eth0')  # ethernet device name
+    port = 13337
+
+    async def send(ep):
+        # recv buffer
+        arr = np.empty(n_bytes, dtype=np.uint8)
+        await ep.recv(arr)
+        assert np.count_nonzero(arr) == 0
+        print("Received NumPy array")
+
+        # increment array and send back
+        arr += 1
+        print("Sending incremented NumPy array")
+        await ep.send(arr)
+
+        await ep.close()
+        lf.close()
+
+    lf = ucp.create_listener(send, port)
+    while not lf.closed():
+        await asyncio.sleep(0.1)
+
+
+
+Process 2 - Client
+~~~~~~~~~~~~~~~~~~
+
+.. code-block:: python
+
+    import asyncio
+    import ucp
+    import numpy as np
+
+    port = 13337
+    n_bytes = 2**30
+    host = ucp.get_address(ifname='eth0')  # ethernet device name
+    ep = await ucp.create_endpoint(host, port)
+    msg = np.zeros(n_bytes, dtype='u1') # create some data to send
```

Comment:
Can we be consistent? Here `dtype='u1'`, in the first process it's `dtype=np.uint8`, I would suggest changing one of them to match the other.